### PR TITLE
Revert to nvtx_kern_sum to get correct profiling info and adding post processing logic

### PR DESCRIPTION
**Pull request by abhinavgoel95**
cc @mgoldfarb-nvidia 
We can add memory kernel times once we figure out how to get that data reliably from nsys

### CODE REVIEW

**Review by hawkinsp — APPROVED** [review]
I didn't review this that closely, but I'm presuming you tested this and found it to work better.

**Comment by hawkinsp** [comment]
mypy is sad, see the lint_and_typecheck result.

**Review by hawkinsp — CHANGES_REQUESTED** [review]
mypy fix needed to merge this.

**Comment by abhinavgoel95** [comment]
I will fix this and ask for another review. Thanks.

File: jax/tools/pgo_nsys_converter.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
   parser = argparse.ArgumentParser(description='Tool to convert NVIDIA Nsys Profiles to the .pbtxt format')
   parser.add_argument("--profile_path", type=str, help="path to nsys profile")
-  parser.add_argument("--post_process", help="post process pbtxt to get minimum cost value for each instruction", action="store_true")
+  parser.add_argument("--post_process", help="post process pbtxt to get max cost value for each instruction", action="store_true")
```
Comment (abhinavgoel95) [inline jax/tools/pgo_nsys_converter.py:31]: @abhinavgoel95  to make this default true.

**Comment by abhinavgoel95** [comment]
@hawkinsp where can I see the mypy/linter errors?

**Comment by hawkinsp** [comment]
I just reapproved the CI runs (unless you're a collaborator on the repository it needs approval to run CI  — I'm surprised you're not, actually, let me know if you want to be added). Lint/mypy errors should show up now.

(You can reproduce these locally by using `pre-commit` in your source tree.)

**Comment by abhinavgoel95** [comment]
@hawkinsp yeah, would like to become a collaborator. 
Fixed the mypy issues, could you re-approve the CI run?

**Pull request merged**